types/configurable/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for configurable 0.0
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Vilim Stubičan <https://github.com/jewbre>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+// Make any object configurable
+export default function(obj: any): Configurable;
+
+export class Configurable {
```

Comment:
I do not think this needs ot be a class..  it should be an interface or a type alais

Reply:
I wasn't really sure due to its usage in other package where it is extending `prototype`. It makes more sense to be an interface.

Changed.